src/utils.ts

```diff
@@ -15,16 +16,17 @@ const constructFilePath = ({ consumerName, providerName }: PactConfigType) =>
 export const writePact = (
   intercept: Interception | XHRRequestAndResponse,
   testCaseTitle: string,
-  pactConfig: PactConfigType
+  pactConfig: PactConfigType,
+  blocklist: string[] = []
 ) => {
```

Comment:
Hey just looking at the interface, on https://github.com/pactflow/cypress-pact/pull/2/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R66
 
 There are very similar but not quite (it adds `content`), and uses unnamed params.

 ```
 const constructPactFile = (
  intercept: Interception | XHRRequestAndResponse,
  testTitle: string,
  pactConfig: PactConfigType,
  blocklist: string[] = [],
  content?: any
)
``` 
 
 would it make sense to make an interface
 
 ```
 Interface {
  intercept: Interception | XHRRequestAndResponse,
  testCaseTitle: string,
  pactConfig: PactConfigType
  blocklist: string[]
}
 ```
 
 Is it worth creating an interface this?
 
Also looking at the code below, I am trying to work out why we would want a scenario where we might expect no `content` and in `constructPactFile` create a pact file with just the default `pactSkeletonObject`.

Reply:
Yep, I'm going to refactor parameter list to a typed parameter object.